uber/utils.py

```diff
@@ -678,6 +680,56 @@ def remove_opt(opts, other):
     return ','.join(map(str, set(opts).difference(other)))
 
 
+def _server_to_url(server):
```

Comment:
I actually wanted to put these in api.py, as that made the most sense, but that caused an import loop that I wasn't able to fix. Whoops!